ivadomed/visualize.py

```diff
@@ -128,7 +129,7 @@ def convert_labels_to_RGB(grid_img):
     return rgb_img
 
 
-def save_tensorboard_img(writer, epoch, dataset_type, input_samples, gt_samples, preds, is_three_dim=False):
+def save_wandb_img(writer, epoch, dataset_type, input_samples, gt_samples, preds, is_three_dim=False):
```

Comment:
FYI, we MAY eventually abstract this... because I suspect we MAY hear push backs when tensor board feedback is gutted, so if that happens, we may need to generalize this portion of the code to handle multiple streams (e.g. Wandb+tensorboard)

But for this PR, probably not need to be too concerned as it is.

Reply:
Strongly agree. Nice one. 👍